demo/src/MemoryModelsUserInput.tsx

```diff
@@ -195,52 +180,47 @@ export default function MemoryModelsUserInput(
-                            setTextData={props.setTextData}
-                            setFailureBanner={props.setFailureBanner}
-                            jsonResult={props.jsonResult}
-                        />
-                        <MemoryModelsTextInput
-                            textData={props.textData}
-                            setTextData={props.setTextData}
-                        />
-                        <Accordion>
-                            <AccordionSummary
-                                expandIcon={<ExpandMore />}
-                                data-testid="rendering-options-accordion"
-                            >
-                                Rendering Options
-                            </AccordionSummary>
-                            <AccordionDetails>
-                                <Card color="neutral" variant="outlined">
-                                    <MemoryModelsConfigInput
-                                        configData={props.configData}
-                                        setConfigData={props.setConfigData}
-                                    />
-                                </Card>
-                            </AccordionDetails>
-                        </Accordion>
-                    </Card>
-                    <Tooltip title="Input JSON to draw diagram">
-                        <span>
-                            <Button
-                                type="submit"
-                                data-testid="input-submit-button"
-                                variant="contained"
-                                color="primary"
-                                disabled={!props.textData}
-                                style={{ textTransform: "none" }}
-                            >
-                                Draw Diagram
-                            </Button>
-                        </span>
-                    </Tooltip>
-                </Grid>
-            </Grid>
+            <MemoryModelsFileInput
+                textData={props.textData}
+                setTextData={props.setTextData}
+                setFailureBanner={props.setFailureBanner}
+                jsonResult={props.jsonResult}
+            />
+            <Box sx={{ display: "flex", justifyContent: "space-between" }}>
+                <MemoryModelsSample
+                    setTextData={props.setTextData}
+                    setConfigData={props.setConfigData}
+                    onTextDataSubmit={props.onTextDataSubmit}
+                />
+                <MemoryModelsConfigInput
+                    configData={props.configData}
+                    setConfigData={props.setConfigData}
+                />
+            </Box>
+            <MemoryModelsTextInput
+                textData={props.textData}
+                setTextData={props.setTextData}
+            />
+            <Box sx={{ display: "flex", justifyContent: "space-between" }}>
```

Comment:
This is another good place to use the `Stack` component. (https://mui.com/material-ui/react-stack/)

You can use the `justifyContent` prop. I don't think you should need to manually specify the button widths.